src/main/steps/common/summary/utils.ts

```diff
@@ -92,8 +92,8 @@ const setkey = (userCase: Partial<CaseWithId>, key: string, language: string | u
       }
       break;
     case 'citizenUserDateOfBirthText':
-      if (userkey === 'Invalid Date' && language === 'cy') {
-        return 'Dyddiad Annilys';
+      if (userkey === en.invalidDate && language === 'cy') {
```

Comment:
userkey should not be picked from translation. please make Invalid Date to some constant like invalidDate. English phrases should not be used for condition checking.

Reply:
Done this vivek